pkg/bloomgateway/bloomgateway.go

```diff
@@ -315,21 +327,63 @@ func (g *Gateway) FilterChunkRefs(ctx context.Context, req *logproto.FilterChunk
 		return req.Refs[i].Fingerprint < req.Refs[j].Fingerprint
 	})
 
-	task, resCh, errCh, err := NewTask(tenantID, req)
+	var expectedResponses int
+	swbPerDay := partitionRequest(req)
```

Comment:
what does `swb` stand for here?

Reply:
`seriesWithBounds`. it's the return type's name